lib/index.js

```diff
@@ -0,0 +1,56 @@
+var ids = require('short-id');
```

Comment:
```suggestion
const ids = require('shortid');
```
Prefer const/let over var as well as the above dependency instead of short-id.